lib/tasks_private/vpc.rake

```diff
@@ -0,0 +1,168 @@
+namespace :spec do
+  require 'ibm_cloud_sdk_core'
+  require 'ibm_cloud_resource_controller'
+  require 'ibm_vpc'
+
+  base_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.config.root.to_s, 'spec')
+  cass_dir = File.join(base_dir, 'vcr_cassettes/manageiq/providers/ibm_cloud')
+  spec_dir = File.join(base_dir, 'models/manageiq/providers/ibm_cloud')
+
+  # Prerequisites:
+  # 1. Log in with IBM Cloud CLI i.e. `ibmcloud login -sso`
+  # 2. Place IBM Cloud API key in secrets config
+  desc "Generate refresher VCR"
+  task :refresh_init => :environment do
+    Dir["#{cass_dir}/vpc/cloud_manager/refresher*.yml"].each do |file|
```

Comment:
`Pathname` will let you `glob()` and return an array of pathnames so you can:
```
cass_dir.glob("vpc/cloud_manager/**/*.yml").each(&:delete)
```

(NOTE `&:method` passed to an enumerator will call the method on each item in the enumeration so this is the same as saying `.each { |file| file.delete }`